lib/v2/kindle/radar.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+    'https://www.amazon.com/gp/help/customer/display.html?nodeId=GKMQC26VQQMM8XSW': {
```

Comment:
```suggestion
    'amazon.com': {
```
The key accepts domain name only